js/leaflet-idw.js

```diff
@@ -0,0 +1,354 @@
+/*
+ (c) 2016, Manuel B�r (www.geonet.ch)
```

Comment:
The Leaflet IDW project lives on GitHub: https://github.com/JoranBeaufort/Leaflet.idw

Your answer makes sense - I asked because I was genuinely unsure. :)